docs/user-guide/team-planner/README.md

```diff
@@ -0,0 +1,101 @@
+
+
+
+1. You can create a new team planner using either the **+ Create** button in the top right near your avatar, or by clicking the
+
+    **+ New team planner** button at the bottom of the black menu bar.
+
+2. If you created a team planner before, you can select one of the planners from the list.
+
+3. Team planners with the visibility option **public** selected appear under the **Public Views** heading in the black menu bar. 
+
+4. Team planners with the visibility option "public" unselected appear under the **Private Views** heading in the black menu bar. Note: If **public** is not selected, the calendar is automatically **private** and can only be seen by you, not other members.
+
+   Both **public** and **private** calendars can be added as a **favored** planner, and would appear under the heading **Favorite Views** in the black menu bar.
+
+
+
+## Planning calendar basics
+
+![Overview of the planning calendar basics](planning-calendar-basics.png)
+
+How to navigate the team planner? - The numbers below correspond with the above image:
+
+1. The top header: Click on the top header (where it says **Team planner** in the picture above) to edit the name of your new team planning calendar. The top header is in edit mode by default, so you can just click right on the text. This is where you can give the new team planning calendar its new name. Your new name gets saved automatically by exiting the top header (by clicking somewhere else, for example).
+2. Use the **+ add existing** button to add an existing work package to your view. Note: You cannot create new work packages from the team planner module. Create new work packages in the work package module, then select them in the team planner.
+3. Add a new team member to you planning calendar using the **+ add assignee**.
+3. **Include projects** filter: By default, this filter will only include the current project you are in. You can select additional projects as you see fit by pressing this filter button and check-marking additional projects. The number in the grey bubble inside of this button will indicate the number of projects that are currently selected inside of the filter.
+4. There is a filter button by which you can focus what is displayed on your calendar (for example, only certain team members or work packages).
+5. This switches you team planning calendar to full screen mode.
+6. Saving, renaming and saving a copy (saving as), or deleting can also be accomplished using the **[⋮]** button. This is also where you can modify the visibility options. Select **public** to allow others in your organization that are members of your respective project to see your planning calendar. If **public** is not selected, the calendar is **private** and can only be seen by you, not other members.
+7. By default the planning calendar only shows the current week. Use these buttons to toggle between the 1-week and 2-week view.
+
+## Getting started
+
+- ##### Step 1: Adding team members
```

Comment:
Please don't skip headline levels & remove the minus in front which indicates a (simple) markdown list
```

## Getting started

### Step 1: ...
 ....
### Step 2: ...
 ...

```